src/components/MobileProductActions.tsx

```
@@ -0,0 +1,95 @@
+}) => {
+  return (
+    <div className="fixed bottom-0 left-0 right-0 z-40 lg:hidden">
+      {/* タグ拡張ボタン */}
+      <div className="bg-background/95 backdrop-blur-sm border-t dark:border-gray-800">
+        <button
+          onClick={onOpenTags}
+          className="w-full flex items-center justify-center gap-2 py-3 px-4 text-sm font-medium text-foreground hover:bg-accent transition-colors"
+          aria-label="タグ一覧を開く"
+        >
+          <Tag className="h-4 w-4" />
+          <span>タグ ({tagCount})</span>
```

Biome (2.1.2)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)